File: Sorting Algorithms/Selection Sort.py
# Selection Sort
array_list=list(map(int, input('Enter the elements of arrays: ').split()))

# Selection sort divides the array into two parts: sorted and unsorted
for i, element in enumerate(array_list):
    # finding the minimum of the array_list
    min_element=element
    for el in array_list[i:]:
        if min_element>el:
            min_element=el
            index=array_list.index(el)
    if min_element!=element:
        array_list[i], array_list[index]=array_list[index], array_list[i]
    
print(array_list, sep=' ')

File: Sorting Algorithms/selectionSort.js
// selection sort

const data = [33,21,12,55,8,4];

function selectionSort(data) {
    let minimum;
    let index;
    for(let i=0;i<data.length-1;i++) {
        minimum = data[i];
        index = i;
        for(let j=i+1;j<data.length;j++) {
            if(minimum>data[j]) {
                minimum = data[j];
                index = j;
            }
        }
        if(index != i) {
            [data[i], data[index]] = [data[index], data[i]];
        }
    }
}

selectionSort(data);
console.log(data);